.github/workflows/flutter-desktop-tests.yml

```diff
@@ -82,5 +85,5 @@ jobs:
           clusterName: ${{ env.BAAS_CLUSTER }}
 
       - name: Run tests
-        run: ${{ inputs.os == 'linux' && 'xvfb-run' || '' }} flutter drive -d ${{ inputs.os }} --target=test_driver/app.dart --suppress-analytics --dart-entrypoint-args="" #--verbose #-a="Some test name"
+        run: ${{ inputs.os == 'linux' && 'xvfb-run' || '' }} flutter drive -d ${{ inputs.os }} --target=test_driver/app.dart --suppress-analytics --dart-entrypoint-args="" --debug --no-dds #--verbose -a="Some test name"
```

Comment:
`flutter drive` launches the dart developer service (DDS) and the test app can not be run. 
The workaround is to set `flutter drive –no-dds` .
[[Proposal] add --no-dds argument to flutter drive](https://github.com/flutter/flutter/issues/94548)